sasmodels/modelinfo.py

```diff
@@ -1092,13 +1094,12 @@ class ModelInfo(object):
     #: the model cannot be run in opencl (e.g., because the model passes
     #: functions by reference), then set this to false.
     opencl = None           # type: bool
+    #: True if the model is compiled with C or OpenCL
+    compiled = None         # type: bool
```

Comment:
It might make sense to make this a dataclass and avoid having variables initialised to values outside their type